lib/node_modules/@stdlib/math/base/special/fresnel/lib/fresnel.js

```diff
@@ -0,0 +1,244 @@
+	7.08840045257738576863E5,
+	-2.99181919401019853726E3,
+	0.0
+];
+
+var QS = [
+	6.07366389490084639049E11,
+	2.24411795645340920940E10,
+	4.19320245898111231129E8,
+	5.17343888770096400730E6,
+	4.55847810806532581675E4,
+	2.81376268889994315696E2,
+	1.00000000000000000000E0
+];
+
+// C(x) for small x
+var PC = [
+	9.99999999999999998822E-1,
+	-2.05525900955013891793E-1,
+	1.88843319396703850064E-2,
+	-6.45191435683965050962E-4,
+	9.50428062829859605134E-6,
+	-4.98843114573573548651E-8,
+	0.0
+];
+
+var QC = [
+	1.00000000000000000118E0,
+	4.12142090722199792936E-2,
+	8.68029542941784300606E-4,
+	1.22262789024179030997E-5,
+	1.25001862479598821474E-7,
+	9.15439215774657478799E-10,
+	3.99982968972495980367E-12
+];
+
+// Auxiliary function f(x)
+var PF = [
+	3.76329711269987889006E-20,
+	1.34283276233062758925E-16,
+	1.72010743268161828879E-13,
+	1.02304514164907233465E-10,
+	3.05568983790257605827E-8,
+	4.63613749287867322088E-6,
+	3.45017939782574027900E-4,
+	1.15220955073585758835E-2,
+	1.43407919780758885261E-1,
+	4.21543555043677546506E-1,
+	0.0
+];
+
+var QF = [
+	1.25443237090011264384E-20,
+	4.52001434074129701496E-17,
+	5.88754533621578410010E-14,
+	3.60140029589371370404E-11,
+	1.12699224763999035261E-8,
+	1.84627567348930545870E-6,
+	1.55934409164153020873E-4,
+	6.44051526508858611005E-3,
+	1.16888925859191382142E-1,
+	7.51586398353378947175E-1,
+	1.00000000000000000000E0
+];
+
+// Auxiliary function g(x)
+var PG = [
+	1.86958710162783235106E-22,
+	8.36354435630677421531E-19,
+	1.37555460633261799868E-15,
+	1.08268041139020870318E-12,
+	4.45344415861750144738E-10,
+	9.82852443688422223854E-8,
+	1.15138826111884280931E-5,
+	6.84079380915393090172E-4,
+	1.87648584092575249293E-2,
+	1.97102833525523411709E-1,
+	5.04442073643383265887E-1,
+	0.0
+];
+
+var QG = [
+	1.86958710162783236342E-22,
+	8.39158816283118707363E-19,
+	1.38796531259578871258E-15,
+	1.10273215066240270757E-12,
+	4.60680728146520428211E-10,
+	1.04314589657571990585E-7,
+	1.27545075667729118702E-5,
+	8.14679107184306179049E-4,
+	2.53603741420338795122E-2,
+	3.37748989120019970451E-1,
+	1.47495759925128324529E0,
+	1.00000000000000000000E0
+];
+
+var polyS = evalrational( PS, QS );
+var polyC = evalrational( PC, QC );
+var polyF = evalrational( PF, QF );
+var polyG = evalrational( PG, QG );
+
+// Array for storing sincos evaluation:
+var sc = [ 0.0, 0.0 ];
+
+
+// MAIN //
+
+/**
+* Computes the Fresnel integrals S(x) and C(x).
+*
+* ## Method
+*
+* Evaluates the Fresnel integrals
+*
+* ```tex
+* \operatorname{S}(x) = \int_0^x \sin\left(\frac{\pi}{2} t^2\right)\,\mathrm{d}t, \\
+* \operatorname{C}(x) = \int_0^x \cos\left(\frac{\pi}{2} t^2\right)\,\mathrm{d}t.
+* ```
+*
+* The integrals are evaluated by a power series for \\( x < 1 \\). For \\( x >= 1 \\) auxiliary functions \\( f(x) \\) and \\( g(x) \\) are employed such that
+*
+* ```tex
+* \operatorname{C}(x) = \frac{1}{2} + f(x) \sin\left( \frac{\pi}{2} x^2 \right) - g(x) \cos\left( \frac{\pi}{2} x^2 \right), \\
```

Comment:
Should these equations be wrapped in an `align`?

Reply:
What's the correct way to align? Does it recognize:

```
C(x) &=& ...
S(x) &=& ...
```

?

Reply:
```
\begin{align}
  f &= x \\
  g &= y
\end{align}
```

Reply:
Added. Though I'm not sure if it'll parse as valid depending on what's going the parsing. (Latex? Mathjax? Katex?)

Reply:
See this [example](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/math/base/dist/gamma/entropy). Ignore the scriptstyle (which I am not sure why it was used). This was rendered via MathJax which is our preferred rendering engine. Should also work with TeX. Katex is not an explicitly supported target.

Reply:
Sounds good. Okay. Have aligned in readme also.

Reply:
Awesome. Thanks!